shared/message/output/CompressedImage.proto

```diff
@@ -33,20 +33,28 @@ message CompressedImage {
             EQUIDISTANT = 2;
             EQUISOLID   = 3;
         }
-
-        Projection projection   = 1;
-        float      focal_length = 2;
-        float      fov          = 3;
-        fvec2      centre       = 4;
-        fvec2      k            = 5;
+        Projection projection = 1;
+        /// Normalised focal length: focal length in pixels / image width
+        float focal_length = 2;
+        /// The angular diameter that the lens covers (the area that light hits on the sensor). In radians
+        float fov = 3;
+        /// Normalised image centre offset: pixels from centre to optical axis / image width
```

Comment:
```suggestion
        /// Normalised image centre offset, in pixels from centre to optical axis / image width
```

Reply:
I think that's less clear. what about this:
```Normalised image centre offset - the number of pixels from centre to optical axis / image width```

Reply:
Fair